src/main/groovy/com/auth0/gradle/oss/extensions/Library.groovy

```diff
@@ -5,4 +5,5 @@ class Library {
     String organization
     String repository
     String description
+    String baselineVersion
```

Comment:
This prop name has a meaning in the context of the japicmp plugin. How about here? Maybe we can come up with a better name that explains that diff check (can't think of any right now)

Reply:
How about `baselineCompareVersion`? It's a little long, but maybe is clearer?

Reply:
Yes, I agree it's clearer and I don't mind it being long as long if it's descriptive. Let's change it 👍

Reply:
New commit to rename to `baselineCompareVersion` pushed. Will also update the PR description with the new name.